templates/CampbellAfrica-v1-template.json

```diff
@@ -0,0 +1,87 @@
+{
+    "conformsTo": "csv2bufr-template-v2.json",
+    "metadata": {
+        "label": "Surface land stations (Campbell - Africa)",
+        "description": "Mapping template for converting CSV data from Campbell data loggers configured and deployed in Africa to BUFR seqeunce 301150, 307080. Template based on synop-template (c754dbb9-ee5e-4346-8b4b-3b9c1ba43290).",
```

Comment:
Rather than adding a new template -and- referring to the old one in the description, can we please remove synop-template (c754dbb9-ee5e-4346-8b4b-3b9c1ba43290)